jms/src/test/scala/akka/stream/alpakka/jms/JmsProducerRetrySpec.scala

```diff
@@ -59,7 +62,10 @@ class JmsProducerRetrySpec extends JmsSpec {
       server.stop() // crash.
 
       Thread.sleep(1000)
-      server.restart() // recover.
+      server.stop()
+      // https://activemq.apache.org/how-do-i-restart-embedded-broker.html
+      server.service.waitUntilStopped()
+      server.start(true) // recover.
```

Comment:
Thank you for modernising this, I believe one `server.stop()` can be removed.

Reply:
I will try.